erpnext/agriculture/doctype/destroyed_plant_log/destroyed_plant_log.json

```diff
@@ -0,0 +1,100 @@
+{
+ "autoname": "DPL.#####",
+ "creation": "2020-09-17 10:37:41.370040",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "plant_batch",
+  "destroy_count",
+  "reason",
+  "actual_date",
+  "amended_from"
+ ],
+ "fields": [
+  {
+   "fieldname": "plant_batch",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Plant Batch",
+   "options": "Plant Batch",
+   "reqd": 1
+  },
+  {
+   "fieldname": "destroy_count",
+   "fieldtype": "Int",
+   "in_list_view": 1,
+   "label": "Destroy Count",
+   "reqd": 1
+  },
+  {
+   "fieldname": "actual_date",
+   "fieldtype": "Date",
+   "label": "Actual Date"
+  },
+  {
+   "fieldname": "amended_from",
+   "fieldtype": "Link",
+   "label": "Amended From",
+   "no_copy": 1,
+   "options": "Destroyed Plant Log",
+   "print_hide": 1,
+   "read_only": 1
+  },
+  {
+   "fieldname": "amended_from",
+   "fieldtype": "Link",
+   "label": "Amended From",
+   "no_copy": 1,
+   "options": "Destroyed Plant Log",
+   "print_hide": 1,
+   "read_only": 1
+  },
```

Comment:
- Fields are duplicated.
- If the DocType is submittable the system will add `amended_from` automatically.